bootstrap

```diff
@@ -47,7 +47,8 @@ Configuration:
     --disable-avx2                  disables use of AVX2 instructions
     --disable-webp                  disables building of webp library and linkage test
     --disable-vcpkg                 disables use of vcpkg for downloading and building dependencies
-    --enable-static-tiledb          enables building TileDB as a static library
+    --enable-static-tiledb          enables building TileDB as a static library (this is the default)
+    --enable-shared-tiledb          enables building TileDB as a shared library
```

Comment:
It's a small point, but I would prefer this option as `linkage=static` or `linkage=shared`.
What you've got here is has no error checking (can't do both at once like the option structure might imply) and is overly verbose.